tests/puppeteer/pages/BO/advancedParameters/team/addProfile.js

```diff
@@ -0,0 +1,31 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class AddProfile extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitleCreate = 'Add new profile • prestashop';
```

Comment:
```suggestion
    this.pageTitleCreate = 'Add new profile •';
```
`prestashop` depends on shopname